specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -13221,6 +16065,111 @@
         }
       }
     },
+    "FunctionEnvelope": {
+      "description": "Web Job Information.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "FunctionEnvelope resource specific properties",
+          "properties": {
+            "name": {
+              "description": "Function name.",
+              "type": "string"
+            },
+            "functionAppId": {
```

Comment:
@ Eric if this is indeed readonly you can annotate it with [PropertySettings(Accessibility = PropertyAccessibility.ReadOnly)] in the model and regenerate the swagger.

---
In reply to: [134406926](https://github.com/Azure/azure-rest-api-specs/pull/1551#discussion_r134406926) [](ancestors = 134406926)